demomultistoreform/Resources/install.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS PREFIX_content_block (id_content_block INT AUTO_INCREMENT NOT NULL, title VARCHAR(255) NOT NULL, description LONGTEXT NOT NULL, enable TINYINT(1) NOT NULL, PRIMARY KEY(id_content_block)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB;
+CREATE TABLE IF NOT EXISTS PREFIX_content_block_shop (id_content_block INT NOT NULL, id_shop INT NOT NULL, INDEX IDX_80518F36D6F01295 (id_content_block), INDEX IDX_80518F36274A50A0 (id_shop), PRIMARY KEY(id_content_block, id_shop)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB;
```

Comment:
In Core SQL statements we do not add indexed and foreign keys, I don't know if we should do likewise for modules 🤔 

Wdyt @PierreRambaud @zuk3975 @kpodemski

Reply:
I don't see any reason to not use them inside modules 🤔

Reply:
If Its not necessary for this module to work, then I would say we could avoid using everything "not default" like index.
Else the user will raise more questions like "how do I generate this `80518F36D6F01295` number for index? Is it necessary for module to work?`

Reply:
On the core, I wouldn't even need these queries, it would be done by doctrine.

But in a module, I do need those foreign keys, I replicated the queries generated by doctrine here, I can't use doctrine at the installation of the module because those are new entities